### PR TITLE
feat: new designs for individual Feed pages

### LINKED ISSUE #464: Final GTFS Schedule feed page: Bounding Box, Feed Summary, and Header
### Describe the problem

[Final design for the Mobility Database GTFS Schedule page is here](https://www.figma.com/design/LZVQecGplb4Lnuh7a24Jef/GTFS-Tool-Forms?node-id=1370-1563&m=dev)

### Proposed solution

Please style the following sections to match the composite: 

- [ ] Header (From the breadcrumbs at the top of the grey section of the page to the Download Latest and See License buttons) 
- [ ] Bounding box component
- [ ] Feed Summary

Cypress tests should be included for these sections within the PR. 

### Alternatives you've considered

_No response_

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by indraneel**
**Summary:**

- closes #464  #465  #466 #490
- Schedule feed page (bounding box, feed summary, header) and realtime feed page (feed summary + related schedule feeds)

Screen Shot 2024-06-25 at 3 30 38 PM
image
Screen Shot 2024-06-25 at 3 21 32 PM

**Testing tips:**

Navigate to an individual feed page to see the new designs reflected.

Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] Add or update any needed documentation to the repo
- [x] Format the title like "feat: [new feature short description]". Title must follow the Conventional Commit Specification(https://www.conventionalcommits.org/en/v1.0.0/).
- [x] Linked all relevant issues
- [x] Include screenshot(s) showing how this pull request works and fixes the issue(s)

### CODE REVIEW

**Comment by Alessandro100** [comment]
Screenshot 2024-06-27 at 09 15 38

Can you add some margin to the chips and header, and align the buttons to the left to match the rest

File: web-app/src/app/screens/Feed/FeedSummary.tsx [hunk 1]
```diff
@@ -2,140 +2,173 @@ import * as React from 'react';
...
-                sx={{ fontSize: 14, fontWeight: 'bold', border: 'none' }}
-              >
-                HTTP Auth Parameter:
-              </TableCell>
-              <TableCell sx={{ border: 'none' }}>
-                {feed?.source_info?.api_key_parameter_name !== null
-                  ? feed?.source_info?.api_key_parameter_name
-                  : 'N/A'}
+              <TableCell sx={{ fontSize: 14, border: 'none' }}>
+                <b>Data type:</b>
```
Comment (Alessandro100) [inline web-app/src/app/screens/Feed/FeedSummary.tsx:104]: We should use 
```
<Typography variant="h6" gutterBottom>
        h6. Heading
      </Typography>
```
that or `subtitle2`

File: web-app/src/app/screens/Feed/FeedSummary.tsx [hunk 2]
```diff
@@ -2,140 +2,173 @@ import * as React from 'react';
...
-              </TableCell>
-              <TableCell sx={{ border: 'none' }}>
-                {feed?.locations !== undefined
-                  ? Object.values(feed?.locations[0])
-                      .filter((v) => v !== null)
-                      .reverse()
-                      .join(', ')
-                  : ''}
+              <TableCell sx={{ fontSize: 14, border: 'none' }}>
+                <b>Producer download URL:</b>
```
Comment (Alessandro100) [inline web-app/src/app/screens/Feed/FeedSummary.tsx:60]: same comment: `Typography`

File: web-app/src/app/screens/Feed/FeedSummary.tsx [hunk 3]
```diff
@@ -2,140 +2,173 @@ import * as React from 'react';
...
-              >
-                Data type:
-              </TableCell>
-              <TableCell sx={{ border: 'none' }}>
-                <Button sx={{ textOverflow: 'ellipsis' }} variant='outlined'>
-                  {feed?.data_type === 'gtfs' && 'GTFS'}
-                  {feed?.data_type === 'gtfs_rt' && 'GTFS Realtime'}
-                </Button>
+              <TableCell sx={{ fontSize: 14, border: 'none' }}>
+                <b>Location:</b>
```
Comment (Alessandro100) [inline web-app/src/app/screens/Feed/FeedSummary.tsx:47]: `Typography`

File: web-app/src/app/screens/Feed/FeedSummary.tsx [hunk 4]
```diff
@@ -2,140 +2,173 @@ import * as React from 'react';
...
+                </div>
               </TableCell>
             </TableRow>
-            {feed?.data_type === 'gtfs' && (
+            {feed?.data_type === 'gtfs' &&
+              feed?.feed_contact_email !== undefined &&
+              feed?.feed_contact_email.length > 0 && (
+                <TableRow>
+                  <TableCell sx={{ fontSize: 14, border: 'none' }}>
+                    <b>Feed contact email:</b>
```
Comment (Alessandro100) [inline web-app/src/app/screens/Feed/FeedSummary.tsx:124]: `Typography`

File: web-app/src/app/screens/Feed/PreviousDatasets.tsx [hunk 5]
```diff
@@ -27,85 +29,153 @@ export default function PreviousDatasets({
...
+      </Typography>
+      <Typography>
+        The Mobility Database fetches and stores new datasets once a week, at
+        11:59 EST on Sundays.{' '}
+      </Typography>
+      <Typography>
+        {datasets !== undefined && datasets.length > 0 && (
+          <div>
+            1-{datasets.length < 10 ? datasets.length : 10} of {datasets.length}{' '}
+            {'Datasets >'}
```
Comment (Alessandro100) [inline web-app/src/app/screens/Feed/PreviousDatasets.tsx:44]: We should use the `ChevronRight` icon over `>` text

**Comment by indraneel** [comment]
Resolved all above `Typography` and `Chevron` nits @Alessandro100

Also:

> Screenshot 2024-06-27 at 09 15 38
> 
> Can you add some margin to the chips and header, and align the buttons to the left to match the rest

Sure, changed: 
image

**Review by Alessandro100 — APPROVED** [review]
Nice stuff, looks good! I like the way you structured the code

**Comment by indraneel** [comment]
Added ability to get data about related Schedule feeds on Realtime Feed pages  -- @Alessandro100 I know you approved but if you want to take a look
image

File: web-app/src/app/screens/Feed/AssociatedFeeds.tsx [hunk 6]
```diff
@@ -0,0 +1,69 @@
...
+    <TableRow key={assocFeed?.id}>
+      <TableCell>{assocFeed.feed_name}</TableCell>
+      <TableCell>
+        {assocFeed.latest_dataset?.downloaded_at !== undefined && (
+          <span style={{ display: 'flex' }}>
+            <a href={`/feeds/${assocFeed?.id}`} rel='noreferrer'>
+              Last updated on{' '}
+              {new Date(assocFeed.latest_dataset?.downloaded_at).toDateString()}
+            </a>
+            <OpenInNewOutlined />
```
Comment (Alessandro100) [inline web-app/src/app/screens/Feed/AssociatedFeeds.tsx:37]: [suggestion] I would include the icon in the `a` tag, that or make the entire row clickable

Comment (indraneel) [reply web-app/src/app/screens/Feed/AssociatedFeeds.tsx:37]: great idea -- done!

**Pull request merged**